docs/user_guide/performance.rst

```diff
@@ -0,0 +1,40 @@
+
+Build performance and size
+==========================
+
+By default this theme includes all of your documentation links in a collapsible sidebar.
+However, this may slow down your documentation builds considerably if you have
+a lot of documentation pages. This is most common with documentation for projects
+with a large API, which use the ``.. autosummary::`` directive to generate
+API documentation.
+
+To improve the performance of your builds in these cases, first try modifying the navigation depth in the sidebar (see :ref:`navigation-depth`).
+If that doesn't work, try the fix in the section below.
+
+.. _remove_toctrees:
+
+Selectively remove pages from your sidebar
+------------------------------------------
+
+.. note::
+
+   This and the following sections are useful for sites that have a lot of pages (such as API
+   documentation with a lot of items). These take much longer to build and will have large
+   output sizes because of all the toctree links. These sections help with this problem,
+   ordered from least-to-most drastic.
```

Comment:
"this and the following sections" no longer makes sense, as it's now the last section on this page.  Same goes for "ordered from least to most drastic"